kalite/topic_tools/annotate.py

```diff
@@ -61,8 +61,7 @@ def update_content_availability(content_list, language="en", channel="khan"):
 
             # Databases have been pre-filtered to only contain existing exercises
             # Assume if the assessment items have been downloaded, then everything is hunky dory.
-            if os.path.exists(contentload_settings.KHAN_ASSESSMENT_ITEM_VERSION_PATH):
-                update["available"] = True
+            update["available"] = True
```

Comment:
After reading through the whole `annotate_content_models` code, I think the best solution is to pre-mark exercises as available on the `content-pack-maker` side.